pkg/stdlib/pkgs/convert.cue

```diff
@@ -6,3 +6,25 @@
 	str?: string
 	...
 }
+
+#Timestamp: {
+	#do:       "timestamp"
+	#provider: "convert"
```

Comment:
`convert` is not a good provider name.. Maybe call it `dateutil`?

Reply:
> `convert` is not a good provider name.. Maybe call it `dateutil`?

good idea